packages/java/parser-jvm-core/src/main/java/dev/hilla/parser/models/ClassInfoModel.java

```diff
@@ -33,6 +33,7 @@ public abstract class ClassInfoModel extends AnnotatedAbstractModel
     private List<ClassRefSignatureModel> interfaces;
     private List<MethodInfoModel> methods;
     private PackageInfoModel pkg;
+    private List<PackageInfoModel> ancestors;
```

Comment:
I guess, if we store a list of ancestors, we no longer need a separate `pkg` field. We could just return a first list item. Or we could even remove the `getPackage` method.

Reply:
It depends. In a general API I'd keep both, as the new one is more expensive to compute, so keeping the old one makes sense. But here, if we know that we're going to always use both anyway, your suggestion could be valid. WDYT?

Reply:
I believe that the cost for computation is relatively negligible. Also, we are not using the single ancestor, only all of them in `@NonNullApi`, and this is the only place where we need that computation for now. So I would say that using `List` is preferable here.